cg/server/api.py

```diff
@@ -119,15 +119,17 @@ def submit_order(order_type):
                 content=request_json, file_format=FileFormat.JSON
             ),
         )
-        project: OrderType = OrderType(order_type)
-        order_in: OrderIn = OrderIn.parse_obj(request_json, project=project)
+        project = OrderType(order_type)
+        order_in = OrderIn.parse_obj(request_json, project=project)
 
         result = api.submit(
             project=project,
             order_in=order_in,
             user_name=g.current_user.name,
             user_mail=g.current_user.email,
         )
+        order_service = OrderService(db)
+        order_service.create_order(order_in)
```

Comment:
At some point we would like to replace the ticket fields on Sample and Case with a foreign key right? For that the order creation would need to be moved further into `api.submit` where Samples and Cases are created. Maybe placing it somewhere there would save time in the next PR. Up to you though

Reply:
Also  should not cases be connected to the orders created in this PR?

Reply:
Ideally the endpoint will just make a single call `order_service.create_order(order_request)`. I don't think we should move this call further down the chain, but rather a lot of logic should be reshuffled and probably removed.

Reply:
Oooh I like the sound of that but I fear you might drown in refactoring 😅

Reply:
I think it is worth doing now. It will save us time and headache in the future.

Reply:
Also the entire ordering process is to coupled to LIMS and Susy, so it would be good to rethink this.

Reply:
I'm sensing scope creep! In the interest of getting on with the project I'll leave refactoring to subsequent PR:s.

We probably should implement integration tests for the order endpoint before refactoring it. This logic is too integral and messy for me to just refactor and will be a month long project in itself. I'll set up some refinements to stake out what the order endpoint is supposed to do and how we want to implement it.

Reply:
Sounds like a good plan1

Reply:
https://github.com/Clinical-Genomics/cg/issues/2879